template/.github/workflow/ci.yml

```diff
@@ -0,0 +1,50 @@
+```yaml
+name: CI/CD
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest mypy black isort
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    
+    - name: Lint with flake8
+      run: flake8 .
+    
+    - name: Check imports with isort
+      run: isort --check-only .
+    
+    - name: Format with Black
+      run: black --check .
+    
+    - name: Type check with mypy
```

Comment:
The yaml file is contradicting the README.md. In the readme there is pydocstyle and pylint but here in the actual template flake8 and mypy are used